common/lib/xmodule/xmodule/js/src/html/edit.js

```diff
@@ -167,7 +155,13 @@
            */
           init_instance_callback: this.initInstanceCallback,
           browser_spellcheck: true
-        });
+        };
+        this.tiny_mce_textarea = $(".tiny-mce", this.element).tinymce(Object.assign(tinymceOptions, window.CustomizeFunctionsHook.getTinymceExtraOptions({
```

Comment:
Hi @tranphan-groove ,

Check window.CustomizeFunctionsHook and window.CustomizeFunctionsHook.getTinymceExtraOptions before using it